buildpack.toml

```
@@ -41,6 +41,12 @@ api = "0.7"
     description = "the application context path"
     name = "BP_TOMCAT_CONTEXT_PATH"
 
+  [[metadata.configurations]]
+    build = true
+    default = "false"
+    description = "Disable the EnvironmentPropertySource for reading environment variables into tomcat config files"
+    name = "BP_TOMCAT_ENVIRONMENT_PROPERTY_SOURCE_DISABLED"
```

```suggestion
    name = "BP_TOMCAT_ENV_PROPERTY_SOURCE_DISABLED"
```

How do you feel about this? Shorten it up a little bit.